lib/travis/api/app/endpoint/builds.rb

```diff
@@ -42,7 +42,7 @@ class Builds < Endpoint
           status 422
           respond_with json
         else
-          payload = { id: params[:id], user_id: current_user.id, source: 'api' }
+          payload = { id: params[:id], user_id: current_user.id, source: 'api', reason: "Build Cancelled manually by User with id: #{current_user.login}" }
```

Comment:
@shahzaibawan current_user.login is not an id, so let's rephrase the message to something like
 "Build Cancelled manually by User: #{current_user.login}".